protelis/protelis-lang/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
```

Comment:
Nope.
The build system is now in kts format, that is the point where the plugin should get applied. Do not regenerate dead files, please.
This is a no-go.